samples/characterization/randomized-benchmarking/Inference.qs

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples {
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Random;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Characterization;
+    open Microsoft.Quantum.Synthesis;
+
+    newtype SmcApproximation = (
```

Comment:
It would be helpful to add some documentation to this type and its fields.

Reply:
Good point, let me add some more comments to this and other parts of this file.